base/testfiles/github-1675.tlg

```diff
@@ -0,0 +1,17 @@
+This is a generated file for the LaTeX2e validation system.
+Don't change this file in any respect.
+> \thesection=\long macro:
+->\thechapter .\@arabic \c@section .
+l. ...\show\thesection
+> \thesection=macro:
+->\thechapter .\arabic {section}.
+l. ...\show\thesection
+> \thesection=macro:
+->\arabic {\blub }.
```

Comment:
as @muzimuzhi  commented: I think counterwithout should expand in a similar way, and not create an `\arabic{\blub}`.